Workshop/GetStarted.md

```diff
@@ -22,6 +22,15 @@ After a few seconds, your new repository should be ready for you.
 | ![image](https://github.com/microsoft/AL-Go/assets/10775043/11d7f51d-f38c-4163-a929-a55f2360911d) |
 |-|
 
+Navigate to the repository, go to the Settings tab and select Actions from the sidebar.
+
+<img width="615" alt="image" src="https://github.com/user-attachments/assets/c31b601c-73e7-4908-98de-32b6529145fc" />

```

Comment:
Use .md image tag instead of html.

| ![image](https://github.com/user-attachments/assets/c31b601c-73e7-4908-98de-32b6529145fc) |
|-|